app-misc/lirc/lirc-0.10.2-r1.ebuild

```diff
@@ -0,0 +1,186 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+
+inherit autotools linux-info python-single-r1 xdg-utils
+
+DESCRIPTION="decode and send infra-red signals of many commonly used remote controls"
+HOMEPAGE="https://www.lirc.org/"
+
+LIRC_DRIVER_DEVICE="/dev/lirc0"
+
+MY_P=${PN}-${PV/_/-}
+S="${WORKDIR}/${MY_P}"
+
+if [[ ${PV} == *_pre* ]] ; then
+	SRC_URI="https://www.lirc.org/software/snapshots/${MY_P}.tar.bz2"
+elif [[ ${PV} == *_p* ]] ; then
+	inherit autotools
+	SRC_URI="https://downloads.sourceforge.net/lirc/${PN}-$(ver_cut 1-3).tar.bz2"
+	SRC_URI+=" mirror://debian/pool/main/l/${PN}/${PN}_$(ver_cut 1-3)-$(ver_cut 5-).debian.tar.xz"
+	S="${WORKDIR}"/${PN}-$(ver_cut 1-3)
+else
+	SRC_URI="https://downloads.sourceforge.net/lirc/${MY_P}.tar.bz2"
+fi
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
+IUSE="audio +devinput doc ftdi gtk inputlirc selinux static-libs systemd +uinput usb X"
+
+REQUIRED_USE="
+	${PYTHON_REQUIRED_USE}
+	gtk? ( X )
+"
+
+COMMON_DEPEND="
+	${PYTHON_DEPS}
+	audio? (
+		>media-libs/portaudio-18
+		media-libs/alsa-lib
+	)
+	$(python_gen_cond_dep '
+		dev-python/pyyaml[${PYTHON_USEDEP}]
+	')
+	ftdi? ( dev-embedded/libftdi:0 )
+	systemd? ( sys-apps/systemd )
+	usb? ( virtual/libusb:0 )
+	X? (
+		x11-libs/libICE
+		x11-libs/libSM
+		x11-libs/libX11
+	)
+"
+
+DEPEND="
+	${COMMON_DEPEND}
+	dev-libs/libxslt
+	$(python_gen_cond_dep '
+		dev-python/setuptools[${PYTHON_USEDEP}]
+	')
+	doc? ( app-text/doxygen )
+	sys-apps/kmod
+	sys-kernel/linux-headers
+"
+
+RDEPEND="
+	${COMMON_DEPEND}
+	gtk? (
+		x11-libs/vte[introspection]
+		$(python_gen_cond_dep '
+			dev-python/pygobject[${PYTHON_USEDEP}]
+		')
+	)
+	inputlirc? ( app-misc/inputlircd )
+	selinux? ( sec-policy/selinux-lircd )
+"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-0.10.1-runtimedirectory.patch"
+	"${FILESDIR}/${PN}-0.10.2-fix-python-pkg.patch"
+)
+
+MAKEOPTS+=" -j1"
+
+pkg_setup() {
+	use uinput && CONFIG_CHECK="~INPUT_UINPUT"
+	python-single-r1_pkg_setup
+	linux-info_pkg_setup
+}
+
+src_prepare() {
+	default
+
+	# Needed for the fix-python-pkg patch
+	eautoreconf
+
+	# https://bugs.gentoo.org/922209
+	sed -i -e "/^libpython / s|\$(libdir)/python\$(PYTHON_VERSION)|$(python_get_stdlib)|" Makefile.in || die
```

Comment:
Why can't this be done to a proper patch to the .am?

Reply:
No hard reason. Initially it was before eautoreconf was hooked up and then I justified it retroactively by it surviving a driver byer removing that.

Reply:
I don't see why it's still not a patch, but I guess let's get it in.